clang/test/Driver/module-fgen-reduced-bmi.cppm

```diff
@@ -0,0 +1,53 @@
+// It is annoying to handle different slash direction
+// in Windows and Linux. So we disable the test on Windows
+// here.
+// REQUIRES: !system-windows
+// On AIX, the default output for `-c` may be `.s` instead of `.o`,
+// which makes the test fail. So disable the test on AIX.
+// REQUIRES: !system-aix
+//
+// RUN: rm -rf %t
```

Comment:
redundant `mkdir`

`// RUN: rm -rf %t && split-file %s %t && cd %t`

If Windows backslashes feel annoying, you can add a `cd %t`